### PR TITLE
APPEALS-31618

### PR DESCRIPTION
**Pull request by breedbah**
Resolves [APPEALS-37272](https://jira.devops.va.gov/browse/APPEALS-37272), [APPEALS-31826](https://jira.devops.va.gov/browse/APPEALS-31826), [APPEALS-31618](https://jira.devops.va.gov/browse/APPEALS-31618)

# Description
-Added migration file to include co_host_hearing_link attribute to virtual_hearings.rb
-Added teh co_host_hearing_link to the virtual_hearing_serializer
-Updated the api call for number of hosts returned to 2
-Added the co_host_link to create response
-Updated the client/copy.json text variables to reflect the text in the UI mockup
-Added the co host link to render in hearingLinks.jsx 
-Updated and added tests that needed to be updated

## Acceptance Criteria
- [ ] Code compiles correctly
- [ ] When a veteran is scheduled for a virtual hearing by a HC that has a designation for webex hearings. The co host link populates and is usable by the HC.

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/APPEALS-38494) or list them below

- [ ] For feature branches merging into master: Was this deployed to UAT?

# Frontend
## User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue
Screenshot 2024-01-17 at 1 18 57 PM

# Backend
## Database Changes
*Only for Schema Changes*

* [ ] Add typical timestamps (`created_at`, `updated_at`) for new tables
* [ ] Update column comments; include a "PII" prefix to indicate definite or potential [PII data content](https://github.com/department-of-veterans-affairs/appeals-team/blob/master/caseflow-team/0-how-we-work/pii-handbook.md#what-is-pii)
* [ ] Have your migration classes inherit from `Caseflow::Migration`, especially when adding indexes (use `add_safe_index`) (see [Writing DB migrations](https://github.com/department-of-veterans-affairs/caseflow/wiki/Writing-DB-migrations))
* [ ] Verify that `migrate:rollback` works as desired ([`change` supported functions](https://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method))
* [ ] Perform query profiling (eyeball Rails log, check bullet and fasterer output)
* [ ] For queries using raw sql was an explain plan run by System Team
* [ ] Add appropriate indexes (especially for foreign keys, polymorphic columns, unique constraints, and Rails scopes)
* [ ] Run `make check-fks`; add any missing foreign keys or add to `config/initializers/immigrant.rb` (see [Record associations and Foreign Keys](https://github.com/department-of-veterans-affairs/caseflow/wiki/Record-associations-and-Foreign-Keys))
* [ ] Add `belongs_to` for associations to enable the [schema diagrams](https://department-of-veterans-affairs.github.io/caseflow/task_trees/schema/schema_diagrams) to be automatically updated
* [ ] Document any non-obvious semantics or logic useful for interpreting database data at [Caseflow Data Model and Dictionary](https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Data-Model-and-Dictionary)

# Best practices
## Code Documentation Updates
- [ ] Add or update code comments at the top of the class, module, and/or component.

### Test Coverage
Did you include any test coverage for your code? Check below:
- [ ] RSpec
- [ ] Jest
- [ ] Other

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [ ] No new code climate issues added

### CODE REVIEW

File: db/migrate/20240117030132_add_co_host_link_to_virtual_hearings.rb
Comment (msteele96) [inline db/migrate/20240117030132_add_co_host_link_to_virtual_hearings.rb:1]: Inherit from Caseflow::Migration

File: client/app/hearings/components/details/HearingLinks.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const VirtualHearingLinkDetails = ({
   isVirtual,
   user,
   label,
-  virtualHearing
+  virtualHearing,
```
Comment (msteele96) [inline client/app/hearings/components/details/HearingLinks.jsx:27]: Is this comma necessary?

Comment (breedbah) [reply client/app/hearings/components/details/HearingLinks.jsx:27]: Resolved

File: db/migrate/20240117030132_add_co_host_link_to_virtual_hearings.rb
Comment (breedbah) [reply db/migrate/20240117030132_add_co_host_link_to_virtual_hearings.rb:1]: Updated

**Review by mchbidwell — APPROVED** [review]

**Pull request merged**